site.yml

```
@@ -21,460 +21,460 @@ Documents:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-ruby-with-minio.md
         Slug: how-to-use-aws-sdk-for-ruby-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for Ruby with MinIO Server - Cookbook/Recipe
+        SEO Description: aws-sdk is the official AWS SDK for the Ruby programming language. In this recipe we will learn how to use aws-sdk for Ruby with MinIO server
+        SEO Keywords: AWS SDK for Ruby, AWS SDK for Ruby on Rails
 
       - How to use AWS SDK for Python with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-python-with-minio.md
         Slug: how-to-use-aws-sdk-for-python-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for Python with MinIO Server - Cookbook/Recipe
+        SEO Description: aws-sdk-python is the official AWS SDK for the Python programming language. In this recipe we will learn how to use aws-sdk-python with MinIO server.
+        SEO Keywords: AWS SDK for Python
 
       - How to use AWS SDK for JavaScript with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-javascript-with-minio.md
         Slug: how-to-use-aws-sdk-for-javascript-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for JavaScript with MinIO Server - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to use aws-sdk for Javascript with MinIO server. aws-sdk is the official AWS SDK for the Javascript programming language.
+        SEO Keywords: AWS SDK for JavaScript
 
       - How to run multiple Minio servers with Træfɪk:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/multiple-minio-servers-with-traefik.md
         Slug: how-to-run-multiple-minio-servers-with-traef-k
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to run multiple MinIO servers with Træfɪk - Cookbook/Recipe
+        SEO Description: This cookbook will explain how you can setup multiple MinIO instances via Docker which you can access on different subdomains through Træfɪk.
+        SEO Keywords: Træfɪk, reverse proxy, golang
 
       - How to use AWS SDK for Go with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-go-with-minio.md
         Slug: how-to-use-aws-sdk-for-go-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for Go with MinIO Server - Cookbook/Recipe
+        SEO Description: aws-sdk-go is the official AWS SDK for the Go programming language. In this recipe we will learn how to use aws-sdk-go with MinIO server
```

Similar to my other comment for the PHP SDK, we might want to have something the following, as the first sentence doesn't show the purpose of the document directly. Once again, please feel free to dismiss if that's not the case.

```suggestion
        SEO Description: In this recipe we will learn how to use aws-sdk-go, the official AWS SDK for the Go programming language, with MinIO server
```